game.py

```diff
@@ -14,30 +14,36 @@
 
 
 def snowman(snowman_word):
-    """Complete the snowman function
-    replace "pass" below with your own code
-    It should print 'Congratulations, you win!'
-    If the player wins and, 
-    'Sorry, you lose! The word was {snowman_word}' if the player loses
-    """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    wrong_guesses_count = 0
+
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        guessed_letter = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
```

Comment:
This line breaks the PEP8 guidelines for line length (79 characters or less including indentation https://peps.python.org/pep-0008/#maximum-line-length). From the linked guide:

> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. 

There are other options than my example below to break up a long line, but what the recommendation above means is that we could break up the line by doing something like:
```python
guessed_letter = get_letter_from_user(
    correct_letter_guess_statuses, 
    wrong_guesses_list
)
```